sway_libs/src/ownership/ownable.sw

```diff
@@ -0,0 +1,50 @@
+library ownable;
+
+dep errors;
+dep events;
+dep ownable_storage;
+
+use errors::AccessError;
+use events::{OwnershipRenounced, OwnershipSet, OwnershipTransferred};
+use ownable_storage::OWNER;
+use std::{auth::msg_sender, hash::sha256, logging::log, storage::{get, store}};
+
+#[storage(read)]
+pub fn only_owner() {
+    let owner = get::<Option<Identity>>(OWNER);
+    require(owner.is_some() && msg_sender().unwrap() == owner.unwrap(), AccessError::NotOwner);
```

Comment:
I think the 2nd condition here is sufficient.

Reply:
Correct me if I'm wrong but I believe this is needed. If the owner is `None` (In the cases of initialized or renounced) then we cannot unwrap it